src/quo2/components/settings/settings_list/view.cljs

```diff
@@ -0,0 +1,133 @@
+(ns quo2.components.settings.settings-list.view
+  (:require [quo2.components.settings.settings-list.style :as style]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.selectors.selectors.view :as selectors]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]))
+
+(defn settings-title
+  [title]
+  [rn/view
+   {:flex 1}
+   [text/text
+    {:accessibility-label :setting-item-name-text
+     :ellipsize-mode      :tail
+     :style               (style/title)
+     :number-of-lines     1
+     :weight              :medium
+     :size                :paragraph-1}
+    title]
+  ])
+
+(def browser-context-icon
+  [rn/view
+   [quo2.icons/icon :browser-context
+    {:container-style style/icon
+     :color           (colors/theme-colors
+                       colors/neutral-50
+                       colors/neutral-40)}]])
+
+(def chevron-icon
+  [rn/view
+   [quo2.icons/icon :chevron-right
+    {:color (colors/theme-colors
+             colors/neutral-50
+             colors/neutral-40)}]])
+
+(defn switch-button
+  [{:keys [:checked?
+           on-change]}]
+  [selectors/toggle
+   {:checked?             checked?
+    :on-change            (fn [new-value] (on-change new-value))
+    ;; TODO: For some reason this prop is not working on ios simulator, spike a little bit more
+    :ios-background-color (colors/theme-colors
+                           colors/neutral-30
+                           colors/neutral-80)
+    ;; TODO: Ask for colors on the track, there is no definition on the design
+    ;; Colors for false value only work on Android
+    ;; :track-color          {:false (colors/theme-colors
+    ;;                               colors/danger
+    ;;                               colors/danger)
+    ;;                       :true  (colors/theme-colors
+    ;;                               colors/danger
+    ;;                               colors/danger)}
```

Comment:
I believe that since you are using the `selectors/toggle` from quo2, you don't have to worry about this.

Reply:
Right, will delete ios-background-color too.